YOLayout/YOLayout_IMP.m

```diff
@@ -138,11 +138,11 @@ - (CGRect)setSize:(CGSize)size inRect:(CGRect)inRect view:(id)view options:(YOLa
     size = sizeThatFits;
   }
   
-  if ((options & YOLayoutOptionsSizeToFitHorizontal) == 0) {
+  if ((options & YOLayoutOptionsSizeToFitHorizontal) == 0 && originalSize.width > 0) {
```

Comment:
Removing the resetting to originalSize (lines 141-147) is the same as removing `YOLayoutOptionsSizeToFitHorizontal` and `YOLayoutOptionsSizeToFitVertical` right? I've found it very useful to have those separate from `YOLayoutOptionsSizeToFit`.

I'll put together some example code that shows why I don't think this `> 0` check is the best. The root is that I want to be able to set views to 0 width or 0 height but still size in the other dimension if I so choose. I don't think we should take away that flexibility. Use case is to be able to reveal a view along one dimension. Like if I have a label that sizes itself, but I want to initially hide it by setting `frame.size.height = 0`, then have it do an animated vertical reveal to its intended height (as opposed to revealing both horizontally and vertically from `CGSizeZero`).